scripts/datasets/url_checksums/book_corpus.txt

```diff
@@ -0,0 +1 @@
+https://gluonnlp-numpy-data.s3-us-west-2.amazonaws.com/pretrain_corpus/Gutenberg.zip 91e842dc3671ed5a917b7ff6a60f5f87397780e2 461506225
```

Comment:
I will help migrate them to the s3 gluon folder and share back the link.

Reply:
Yes, currently everything is inside another bucket and we should do the merge.